zipline/data/us_equity_minutes.py

```diff
@@ -49,10 +48,29 @@ def write_metadata(directory, first_trading_day):
         json.dump(metadata, fp)
 
 
+def _bcolz_minute_index(trading_days):
+        minutes = np.zeros(len(trading_days) * MINUTES_PER_DAY,
+                           dtype='datetime64[ns]')
+        market_opens = tradingcalendar.open_and_closes.market_open
+        mask = market_opens.index.slice_indexer(start=trading_days[0],
+                                                end=trading_days[-1])
+        opens = market_opens[mask]
+        for i, market_open in enumerate(opens):
+            start = market_open.value
```

Comment:
This is individually unboxing every timestamp object in every iteration of the loop.  It seems like we could straightforwardly do all of this with vectorized numpy operations?